src/control/server/ctl_system.go

```diff
@@ -34,7 +34,7 @@ import (
 	"github.com/daos-stack/daos/src/control/system"
 )
 
-var systemRPCMaxDuration = 30 * time.Second
+var systemRequestTimeout = 30 * time.Second
```

Comment:
Just noticed that this is a var instead of const. Not something to block on, but if you need to make any updates it'd be good to fix this.